### ISSUE TITLE
[BUG] revert to older rustc version

### ISSUE THREAD

**Issue by Horki** [a5huynh/defender-game#9]
Hi @a5huynh , pinning to rustc `1.40` due to errors with newer compiler

please test #8

**Comment by Horki** [comment]
```
thread 'main' panicked at 'attempted to leave type `std::mem::ManuallyDrop<xlib::Xlib>` uninitialized, which is invalid', ~/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/mem/mod.rs:671:9
```

**Comment by a5huynh** [comment]
Closed with #8
